RELEASE.rst

```diff
@@ -10,6 +10,15 @@ Version 0.9.0 (Released October 01, 2019)
 - Recreated submissions zip file when student(s) score reset (#239)
 - Fixed race condition when removing grades SGA (#117)
 
+Version 0.9.0 (Released October 01, 2019)
+-------------
+
+- Possible fix for copying units with SGA (#254)
+- Correct the courseware imports
+- Added focus to file upload error message (#241)
+- Recreated submissions zip file when student(s) score reset (#239)
+- Fixed race condition when removing grades SGA (#117)
+
```

Comment:
Why is this section of the release notes duplicated? Please remove it. 

(Note that updates to the Release notes are handled automatically by our release script, so there shouldn't be any changes to this file at all)

Reply:
That was an accidental change due to previous merge things, it has been removed.